ffi/capi/bindings/dart/AnyCalendarKind.g.dart

```diff
@@ -105,17 +106,23 @@ enum AnyCalendarKind {
   }
 }
 
-@meta.ResourceIdentifier('ICU4XAnyCalendarKind_get_for_locale')
-@ffi.Native<_ResultInt32Void Function(ffi.Pointer<ffi.Opaque>)>(isLeaf: true, symbol: 'ICU4XAnyCalendarKind_get_for_locale')
+@RecordSymbol('ICU4XAnyCalendarKind_get_for_locale')
+@ffi.Native<_ResultInt32Void Function(ffi.Pointer<ffi.Opaque>)>(
+    isLeaf: true, symbol: 'ICU4XAnyCalendarKind_get_for_locale')
 // ignore: non_constant_identifier_names
-external _ResultInt32Void _ICU4XAnyCalendarKind_get_for_locale(ffi.Pointer<ffi.Opaque> locale);
+external _ResultInt32Void _ICU4XAnyCalendarKind_get_for_locale(
```

Comment:
please revert format-only changes. this is raw diplomat-gen output, and diplomat-gen cannot format, as that requires dart to be installed (and stable, which it isn't). if this is an issue in your repo, can we generate some ignore attribute?

Reply:
https://github.com/unicode-org/icu4x/issues/4291

Formatting should happen when code is imported into g3, there are already policies and tools for this.

Reply:
If formatting afterwards is not an issue, then I can just format when copying the code in `package:intl4x`. Is also in line with the discussed option there.